fixtures/altman/ap-150.json

```
@@ -0,0 +1,920 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "AP-150",
+  "categories": ["Color Changer", "Dimmer", "Effect"],
```

```suggestion
  "categories": ["Color Changer"],
```

I don't think strobe/pulse is enough to warrant Effect here -- every LED does those.

I wish OFL had a "PAR" category!